esignet-integration-api/src/main/java/io/mosip/esignet/api/dto/AuthChallenge.java

```diff
@@ -6,11 +6,13 @@
 package io.mosip.esignet.api.dto;
 
 import io.mosip.esignet.api.util.ErrorConstants;
+import io.mosip.esignet.api.validator.Format;
 import lombok.Data;
 
 import javax.validation.constraints.NotBlank;
 
 @Data
+@Format
```

Comment:
This is a class-level validator, and it validates the authFactorType and format mapping. lets change the annotation name @pvsaidurga